packages/embed/src/factories/create-popover/create-popover.spec.ts

```diff
@@ -11,6 +11,17 @@ const mockedLocalStorage = {
 
 beforeAll(() => {
   jest.useFakeTimers('modern').setSystemTime(new Date('2021-06-01 00:00:00').getTime())
+  Object.defineProperty(window, 'matchMedia', {
+    writable: true,
+    value: jest.fn().mockImplementation((query) => ({
+      matches: false,
+      media: query,
+      onchange: null,
+      addEventListener: jest.fn(),
+      removeEventListener: jest.fn(),
+      dispatchEvent: jest.fn(),
+    })),
+  })
```

Comment:
Wow, looks smart, what does it do exactly?

Reply:
we need to mock `matchMedia` because js-dom does not support it